Traceback (most recent call last):
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nest_asyncio.py", line 90, in run_until_complete
    return f.result()
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/asyncio/futures.py", line 201, in result
    raise self._exception
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/asyncio/tasks.py", line 256, in __step
    result = coro.send(None)
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/luo/miniconda3/envs/venv-pyrsimg/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import os
from pyrsimg import readTiff
from pyrsimg import imgShow, imsShow
import matplotlib.pyplot as plt

------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[1], line 2[0m
[1;32m      1[0m [38;5;28;01mimport[39;00m [38;5;21;01mos[39;00m
[0;32m----> 2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m [38;5;28;01mimport[39;00m readTiff
[1;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m [38;5;28;01mimport[39;00m imgShow, imsShow
[1;32m      4[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpyplot[39;00m [38;5;28;01mas[39;00m [38;5;21;01mplt[39;00m

File [0;32m~/Library/CloudStorage/OneDrive-Personal/GitHub/pyrsimg/pyrsimg/__init__.py:7[0m
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m[38;5;21;01m.[39;00m[38;5;21;01mlay_stack[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m
[1;32m      6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m[38;5;21;01m.[39;00m[38;5;21;01mmetric_proc[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m
[0;32m----> 7[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m[38;5;21;01m.[39;00m[38;5;21;01mmetrics[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m
[1;32m      8[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m[38;5;21;01m.[39;00m[38;5;21;01mraster_vec[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyrsimg[39;00m[38;5;21;01m.[39;00m[38;5;21;01mrsimg_io[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m

File [0;32m~/Library/CloudStorage/OneDrive-Personal/GitHub/pyrsimg/pyrsimg/metrics.py:67[0m
[1;32m     63[0m         iou[38;5;241m.[39mappend(iou_score)
[1;32m     64[0m     [38;5;28;01mreturn[39;00m np[38;5;241m.[39mmean(iou)
[0;32m---> 67[0m [38;5;28;01mdef[39;00m [38;5;21mplot_confusion_matrix[39m(cm, classes, normalize[38;5;241m=[39m[38;5;28;01mFalse[39;00m, title[38;5;241m=[39m[38;5;124m'[39m[38;5;124mConfusion matrix[39m[38;5;124m'[39m, cmap[38;5;241m=[39m[43mplt[49m[38;5;241m.[39mcm[38;5;241m.[39mBlues):
[1;32m     68[0m     [38;5;28;01mif[39;00m normalize:
[1;32m     69[0m         cm [38;5;241m=[39m cm[38;5;241m.[39mastype([38;5;124m'[39m[38;5;124mfloat[39m[38;5;124m'[39m) [38;5;241m/[39m cm[38;5;241m.[39msum(axis[38;5;241m=[39m[38;5;241m1[39m)[:, np[38;5;241m.[39mnewaxis]

[0;31mNameError[0m: name 'plt' is not defined
NameError: name 'plt' is not defined

